exercises/binary-search-tree/binary_search_tree_test.py

```diff
@@ -0,0 +1,35 @@
+import unittest
+
+from binary_search_tree import BinarySearchTree
+
+
+class BinarySearchTreeTests(unittest.TestCase):
+    bst = BinarySearchTree()
+
+    def test_add_integer_number1(self):
+        self.assertTrue(self.bst.add(7))
```

Comment:
We use `assertIs(actual, True)` and `assertIs(actual, False)` rather than `assertTrue(actual)` or `assertFalse(actual)` ([#419](https://github.com/exercism/python/pull/419)).

Reply:
Done